rfc-editor/rfc8794.xml

```diff
@@ -2909,82 +2635,45 @@ be appropriate to change the reference to the newer version?
     </author>
     <date year="2011"></date>
   </front>
-<seriesInfo name="ISO/IEC" value="9899:2011" />
+  <seriesInfo name="ISO/IEC" value="9899:2011"></seriesInfo>
 </reference>
-
-<xi:include href="https://xml2rfc.ietf.org/public/rfc/bibxml/reference.RFC.2119.xml"/>
+<xi:include href="https://xml2rfc.ietf.org/public/rfc/bibxml/reference.RFC.0020.xml"/>
+<xi:include href="https://xml2rfc.ietf.org/public/rfc/bibxml/reference.RFC.3339.xml"/>
 <xi:include href="https://xml2rfc.ietf.org/public/rfc/bibxml/reference.RFC.5646.xml"/>
-<xi:include href="https://xml2rfc.ietf.org/public/rfc/bibxml/reference.RFC.7405.xml"/>
-
 <reference anchor="XHTML" target="http://www.w3.org/TR/2018/SPSD-xhtml-basic-20180327/">
   <front>
     <title>XHTML(tm) Basic 1.1 -- Second Edition</title>
-    <author fullname="Shane McCarron" initials="S." surname="McCarron" role="editor">
-      <organization></organization>
+    <author fullname="Shane McCarron" initials="S." surname="McCarron">
+      
     </author>
     <date year="2018" month="March" day="27"></date>
   </front>
-     <seriesInfo name="W3C" value="Recommendation" />
-     <seriesInfo name="Latest version available at" value="http://www.w3.org/TR/xhtml-basic" />
+  <seriesInfo name="Latest version available at" value="http://www.w3.org/TR/xhtml-basic"></seriesInfo>
 </reference>
-
-<xi:include href="https://xml2rfc.ietf.org/public/rfc/bibxml/reference.RFC.8126.xml"/>
-
-<reference anchor="IEEE.754" target="https://standards.ieee.org/standard/754-2019.html">
+<reference anchor="ISO.3309.1979" target="">
   <front>
-    <title>IEEE Standard for Binary Floating-Point Arithmetic</title>
+    <title>Data communication -- High-level data link control procedures -- Frame structure</title>
     <author>
-      <organization>IEEE</organization>
+      <organization>International Organization for Standardization</organization>
     </author>
+    <date year="1979"></date>
   </front>
+  <seriesInfo name="ISO" value="Standard 3309"></seriesInfo>
```

Comment:
I guess, this has been superseded by ISO/IEC 13239:2002.

Reply:
I looked at all RFCs (didn't read them all) and there's no mention of ISO 13239. There are plenty of ISO 3309, so I would rather keep in line. The date and format differ from what we use so I will edit that.

Reply:
okay, thank you for the explanation

Reply:
Actually, looking at the ISO links we use, only the latest version is available for download (after some $ exchange). So we link to something people can't even download. They can still follow links. For example rfc8122 uses https://www.iso.org/standard/64854.html which is not the latest and can't be downloaded.

I still prefer to use the most common names for standards (and also avoid ISO if I can).

Reply:
I agree plenty with you, even having worked many years in ISO TCs. Nobody is perfect…

Reply:
I created a separate PR #383 to clean all the references.